docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -238,6 +238,31 @@ will return a JSON object of the form:
 
 where 123456, 342332, etc are the candidates that exist for this project.
 
+#### 2.1.5 Single project electrophysiology recordings
+```
+GET /projects/$ProjectName/recordings
+```
+
+will return a JSON object of the form:
+```js
+{
+  "Recordings": [
+    {
+      "Candidate":"300167",
+      "PSCID":"OTT167",
+      "Visit":"V1",
```

Comment:
This is inconsistent with the other API endpoints.. they only ones that include PSCID are the candidate endpoints and ones directly related to PSCIDs, and in the other endpoints the Candidate and Visit go in a "Meta: { .. }" attribute

Reply:
technically, the filenames contain the PSCID, not CandID so it feels weird not displaying the PSCID in the recordings endpoint. 

Note: The reason why the PSCID is in the filename is that it was the only way to determine the site of the candidate from the BIDS dataset...